test/unit/session/Business/Service/SessionCookieServiceTest.php

```diff
@@ -204,21 +208,19 @@ private function createSessionCookieAttributeFactoryMock(): SessionCookieAttribu
         private function createSessionCookieAttributeCollection(): SessionCookieAttributeCollection
         {
             return (new SessionCookieAttributeCollection())
-                ->add(new SessionCookieAttribute(self::SESSION_ATTRIBUTE_NAME, self::SESSION_ATTRIBUTE_VALUE));
+                ->add(new SessionCookieAttribute(self::SESSION_ATTRIBUTE_NAME, self::SESSION_ATTRIBUTE_VALUE))
+                ->add(new SessionCookieAttribute('something', 'xyz'))
+                ->add(new SessionCookieAttribute('lifetime', '2'))
+                ->add(new SessionCookieAttribute('path', '/'));
         }
 
         private function expectCookieParametersCall(string $domain, int $lifetime): void
         {
-            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('domain', 'lifetime'));
-            self::setGlobalFunctionExpectations(
-                'session_set_cookie_params',
-                true,
-                $lifetime,
-                $this->createSessionCookieAttributeString(),
-                $this->getCookieDomain($domain),
-                Request::isHttps(),
-                true
-            );
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('lifetime', $lifetime));
```

Comment:
compact function working expecting something like that if i'm not mistaken
```suggestion
            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('lifetime'));
```